.github/workflows/bump_version.yml

```diff
@@ -0,0 +1,58 @@
+name: Bump Version
+
+on:
+  workflow_dispatch:
+    inputs:
+      newVersion:
+        description: "Version name"
+        required: true
+        type: string
+
+jobs:
+  bump_version:
+    name: Bump version
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout the latest code
+        uses: actions/checkout@v2
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Bump Version XML
+        uses: chkfung/android-version-actions@v1.2.1
+        with:
+          gradlePath: sample-xml/app/build.gradle.kts
+          versionCode: ${{ github.run_number }}
+          versionName: ${{ github.event.inputs.newVersion }}
+
+      - name: Bump Version Compose
+        uses: chkfung/android-version-actions@v1.2.1
+        with:
+          gradlePath: sample-compose/app/build.gradle.kts
+          versionCode: ${{ github.run_number }}
+          versionName: ${{ github.event.inputs.newVersion }}
```

Comment:
@doannimble I think we also need to change the version inside `version.properties` 🙏 

Please have a look at this example: https://github.com/nimblehq/android-templates/pull/381/files